library/core/src/main/java/com/google/android/exoplayer2/text/cea/CeaUtil.java

```diff
@@ -33,6 +37,26 @@
   private static final int USER_ID = 0x47413934; // "GA94"
   private static final int USER_DATA_TYPE_CODE = 0x3;
 
+  public static void consume(long presentationTimeUs, ParsableByteArray buffer,
+      TrackOutput[] outputs, int mode) {
+    switch (mode) {
+      case MODE_H262:
+        consumeUserData(presentationTimeUs, buffer, outputs);
+        break;
+      case MODE_H264:
+        consumeSei(presentationTimeUs, buffer, outputs);
```

Comment:
Considering the modes are actually only 2 and they are always known at compile time, why not just have a consumeUserData and consumeSei made public? If so, public methods should be documented.